homeassistant/components/foscam/camera.py

```diff
@@ -17,6 +28,27 @@
 
 FOSCAM_COMM_ERROR = -8
 
+DOMAIN = "foscam"
```

Comment:
Please don't define the `DOMAIN` constant in this module. That's ambiguous since this platform belongs to the camera domain and the integration domain is foscam. It's ok to import it from a const.py module as a different name eg `FOSCAM_DOMAIN`.